### ISSUE TITLE
Cursor becomes invisible

### ISSUE THREAD

**Issue by zhaytee** [Microsoft/vscode#20094]
- VSCode Version: 1.9.0
- OS Version: macOS 10.12.3

Steps to Reproduce:
1. Switch between tabs.

When switching between tabs, the editing cursor becomes invisible. I can usually get it to show up again if I select some text, but even that sometimes doesn't work. I tried setting "editor.disableMonospaceOptimizations" to true, as mentioned in the release notes, but VSCode insists that it's not a valid option (and it doesn't seem to help, anyway).

Happy to provide more information if needed!

**Comment by alexandrudima** [comment]
@zhaytee Can you please add a screen recording of the issue (a gif) ? (i.e. how do you switch between tabs, via mouse or keybindings, how do you select some text, via mouse or keybindings, etc. -- there are many ways to do the same thing that would become immediately obvious with a screen recording)

Also, can you please include your settings.

**Comment by zhaytee** [comment]
@alexandrudima Here's a screencap clearly demonstrating the issue. It only started happening after I installed 1.9, BTW.

![wheres-my-cursor](https://cloud.githubusercontent.com/assets/43938/22693250/4fe5272c-ecf7-11e6-9961-f33876763abf.gif)

My settings:
```
{
    "editor.fontFamily": "Input Mono",
    "editor.fontWeight": "400",
    "editor.fontSize": 15,
    "editor.autoClosingBrackets": false,
    "editor.quickSuggestionsDelay": 500,
    "editor.cursorStyle": "block",
    "editor.renderWhitespace": "boundary",
    "editor.renderControlCharacters": true,
    "window.newWindowDimensions": "maximized"
}
```

**Comment by alexandrudima** [comment]
@zhaytee 

Thank you for the extra info. It is the block cursor issue reported at #19808.

A fix will roll out with the upcoming 1.9.1 release, and a fix is already in the insiders channel.

**Comment by zhaytee** [comment]
Ah, okay! Thanks. 👍🏼

Joe
